src/cr/cube/cube.py

```diff
@@ -231,13 +233,25 @@ def __repr__(self):
         except Exception:
             return super(Cube, self).__repr__()
 
+    @lazyproperty
+    def available_measures(self):
+        """frozenset of available measures in the cube response."""
+        return frozenset(
+            self._cube_response.get("result", {}).get("measures", {}).keys()
+        )
```

Comment:
This needs to be a collection of `Enum` members, not a collection of `str` values. The interface we want to present to callers is:

```python
from cr.cube.enums import MEASURE

if MEASURE.COUNTS in cube.available_measures:
    # --- do the counts thing ---
```

Otherwise we have coupled our clients to the internal schema we use, which is bad juju.